ts/src/phemex.ts

```diff
@@ -16,7 +16,7 @@ export default class phemex extends Exchange {
             'id': 'phemex',
             'name': 'Phemex',
             'countries': [ 'CN' ], // China
-            'rateLimit': 120.5,
+            'rateLimit': 60, // vapi -> 60, original api -> 120
```

Comment:
@AlgoQ which api are we using by default? looking at the `hostname` seems the normal one 🤔

Reply:
Yeah it looks like vapi hostname would be https://vapi.phemex.com/ And I think if we wanted to use vapi base ratelimit it should be 12 not 60 unless there's an error in the docs

Reply:
1000 / (1000/ 60) = 60
1000 / (5000 / 60) = 12